apps/model-registry/upstream/overlays/db/model-registry-db-deployment.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         runAsNonRoot: true
       containers:
       - name: db-container
-        image: mysql:8.3.0
+        image: mysql
```

Comment:
I like the upgrage, but should you not at least say mysql:8 or mysql:9 ?

Reply:
We're still bound the the support of Google MLMD, which is mysql 8.
This will possibly be more under our control after:
- https://github.com/kubeflow/model-registry/issues/865

Reply:
oh I see, you want to sort of default also here?

Reply:
I thought it was sufficient for the kustomize img replacement, let me take care of it in Model Registry directly then 👍

Reply:
https://github.com/kubeflow/model-registry/pull/1137

Reply:
if you don't mind, we will include this in the next sync.